src/main/scala/example/BookBot.scala

```diff
@@ -0,0 +1,48 @@
+package example
+
+import scalaj.http._
+import io.circe._, io.circe.parser._
+import example.models.errors.{Error, HttpError, TransformError}
+
+object BookBot extends App {
+  getISBN("Harry Potter", "Rowling") match {
+    case Left(error) => Console.print(error.getMessage)
+    case Right(value) => Console.print(value)
```

Comment:
I like how you left the Console.print as far to the edge of the app as you could.